src/App.tsx

```diff
@@ -1,7 +1,38 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string | null;
+};
+
+export class App extends Component<{}, State> {
+  state: State = {
+    pressedKey: null,
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.getEvent);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.getEvent);
+  }
+
+  getEvent = (event: KeyboardEvent) => {
```

Comment:
Take better name for this method

setPressedKey or handlePrededKey

it was in previouses checklists